CHANGELOG.md

```diff
@@ -94,7 +94,8 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 - lti:
   - service worker 401 reconnect
-- Add a check on video file size when uploading content
+  - Add a check on video file size when uploading content 
+  - Add a check on deposited file size when uploading content 
```

Comment:
This is still not in the good section

Reply:
Thought I deleted this one but it must come from a rebase error from me